catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/plugin/impl/PolicyResponseImpl.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.plugin.impl;
+
+import java.util.Map;
+import java.util.Set;
+
+import ddf.catalog.plugin.PolicyResponse;
+
+/**
+ * Generic implementation of {@link PolicyResponse}
+ */
+public class PolicyResponseImpl implements PolicyResponse {
```

Comment:
If we don't expect a policy response to be changed once created by a plugin, could we take in the operation and item policies in a constructor and make this class immutable instead of having setters?